frontend/src/bundles/common/components/avatar/avatar.tsx

```diff
@@ -0,0 +1,36 @@
+import { Typography } from '~/bundles/common/components/components.js';
+import { getValidClassNames } from '~/bundles/common/helpers/helpers.js';
+
+import styles from './styles.module.scss';
+
+type Properties = {
+    userFullName: string;
+    url?: string;
+    size?: 'small' | 'medium';
+};
+
+const Avatar: React.FC<Properties> = ({
+    url,
+    size = 'small',
+    userFullName,
+}) => {
+    const avatarClasses = getValidClassNames(styles.avatar, styles[size]);
+
+    const [userName, userSurname] = userFullName.split(' ');
+    const firstLetterIndex = 0;
+    const userNameAbbreviation = userSurname
+        ? `${userName[firstLetterIndex].toUpperCase()}${userSurname[
+              firstLetterIndex
+          ].toUpperCase()}`
+        : `${userName[firstLetterIndex].toUpperCase()}`;
```

Comment:
I think we need to create some general helpers for this logic. Because I already saw it somewhere